doc/man7/provider-kdf.pod

```diff
@@ -318,9 +318,15 @@ OSSL_FUNC_kdf_gettable_params(), OSSL_FUNC_kdf_gettable_ctx_params() and
 OSSL_FUNC_kdf_settable_ctx_params() should return a constant B<OSSL_PARAM>
 array, or NULL if none is offered.
 
+=head1 NOTES
+
+The KDF life-cycle is described in L<life_cycle-kdf(7)>.  Providers should
+ensure that the various transitions listed there are supported.  At some point
+the EVP layer will begin enforcing the listed transitions.
```

Comment:
Wasnt there also the statement - This will not be considered a breaking change

Reply:
It's there in `man3/EVP_KDF.pod`.  I didn't think it needs to be here as well because the provider side will no longer be called once the enforcement is in place.